src/org/anjocaido/groupmanager/GMConfiguration.java

```diff
@@ -165,9 +163,9 @@ public void load() {
 
 			
 
-			Object level = ((Map<String, String>) getElement("settings", GMconfig).get("logging")).get("level"); //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
-			if (level instanceof String)
-				loggerLevel = (String) level;
+			String level = ((Map<String, String>) getElement("settings", GMconfig).get("logging")).get("level"); //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$

```

Comment:
One last issue, this could throw an exception as its possible to get something else than a String. It would mean their config is seriously messed up but it could return all kinds of objects.